src/components/ec-option-card/ec-option-card.vue

```diff
@@ -0,0 +1,199 @@
+  if (to?.value) {
+    return 'router-link';
+  } if (href?.value) {
+    return 'a';
+  }
+  return 'button';
+});
+
+const isOptionCardDisabled = computed(() => ({ disabled: isDisabled?.value }));
+
+const optionCardButtonType = computed(() => (isSubmit?.value ? 'submit' : 'button'));
+
+const optionCardNavigationType = computed(() => (to?.value ? { to: to.value } : { href: href?.value }));
+
+const componentProps = computed(() => {
+  const newProps = {
+    ...isOptionCardDisabled.value,
+    ...(componentTag.value === 'button' && { type: optionCardButtonType.value }),
+    ...optionCardNavigationType.value,
+  };
+
+  return newProps;
+});
+
+const emit = defineEmits(['click', 'submit']);
+
+function getOptionCardClasses() {
+  return {
+    'ec-option-card--disabled': Boolean(isDisabled?.value),
+    'ec-option-card--accent': optionCardType?.value === OptionCardType.OPTION_CARD_ACCENT,
+    'ec-option-card--danger': optionCardType?.value === OptionCardType.OPTION_CARD_DANGER,
+  };
+}
+
+function getOptionIconClass() {
+  return {
+    'ec-option-card__title-img--disabled': Boolean(isDisabled?.value),
+    'ec-option-card__title-img--accent': optionCardType?.value === OptionCardType.OPTION_CARD_ACCENT,
+    'ec-option-card__title-img--danger': !isDisabled?.value && (optionCardType?.value === OptionCardType.OPTION_CARD_DANGER),
+
+  };
+}
+
+function getOptionCaptionClass() {
+  return {
+    'ec-option-card__caption--disabled': Boolean(isDisabled?.value),
+    'ec-option-card__caption--accent': optionCardType?.value === OptionCardType.OPTION_CARD_ACCENT,
+    'ec-option-card__caption--danger': !isDisabled?.value && (optionCardType?.value === OptionCardType.OPTION_CARD_DANGER),
+  };
+}
+
+function setClickAction() {
+  if (componentTag.value === 'button' && !isDisabled?.value) {
+    return optionCardButtonType.value === 'submit' ? emit('submit') : emit('click');
+  }
+  return null;
+}
+</script>
+
+<style>
+:root {
+  --ec-option-card-width: 304px;
+  --ec-option-card-max-height: 80px;
+  --ec-option-card-min-height: 56px;
+}
+
+.ec-option-card {
+  @apply tw-border tw-border-solid tw-rounded tw-border-gray-6;
+  @apply tw-text-left tw-body-text tw-text-gray-3;
+  @apply tw-p-16;
+  @apply tw-flex tw-flex-col;
+
+  width: var(--ec-option-card-width);
+  max-height: var(--ec-option-card-max-height);
+  min-height: var(--ec-option-card-min-height);
+
+  &--accent {
+    @apply tw-text-key-4;
+  }
+
+  &--danger {
+    @apply tw-text-error;
+  }
+
+  &--disabled {
+    @apply tw-pointer-events-none tw-cursor-default tw-bg-gray-7 tw-text-gray-5;
+  }
+
+  &:hover,
+  &:active {
+    @apply tw-no-underline;
+    @apply tw-cursor-pointer;
+    @apply tw-bg-gray-7;
+  }
+
+  &__title-img {
+    @apply tw-mr-8 tw-fill-gray-3;
+
+    &--accent {
+      @apply tw-fill-key-4;
+    }
+
+    &--danger {
+      @apply tw-fill-error;
+    }
+
+    &--disabled {
+      @apply tw-fill-gray-4;
+    }
+  }
+
+  &__caption {
```

Comment:
![image](https://user-images.githubusercontent.com/5939240/220626699-ae22f80d-66c4-405b-a3fe-23a84dd0e522.png)
We should clarify how to deal with long strings in caption. As well as in title too.

Reply:
Oh yes, didn't think of that.... I'll ask Alba in the slack thread...

Reply:
How Alba said we deal with this situation?

Reply:
Alba said to make the box height grow and it doesn't matter if the Option cards have a different height.